src/lib/Scenes/Fair/Fair.tsx

```diff
@@ -106,6 +110,12 @@ export const FairContainer = createFragmentContainer(
       id
       __id
       ...FairDetail_fair
+
+      organizer {
+        website
+      }
+      about
+      ticketsLink
     }
```

Comment:
This gets put in a few places now, it increases the main payload a bit sure - but additional will now be instant thanks to caching.